### ISSUE TITLE
Can the authorizer be made optional, incase I am trying to set this gateway for a public website

### ISSUE THREAD

**Issue by orange-oranje** [awslabs/amazon-apigateway-ingress-controller#18]
I would still need the edge optimisation, throttling, logging but not the authorisation.

It needs to be public

**Comment by branst** [comment]
Agree, removing the annotarion of client arn doesnt make the APIG methods public.

Any chance of implementing this>

**Comment by branst** [comment]
I've forked and eliminated Auth by default: you can compile it at: https://github.com/branst/amazon-apigateway-ingress-controller

**Comment by henrycpainter** [comment]
+1 for this.  Use case is to handle Cognito-based authentication at the load balancer.
